src/utils/api.ts

```diff
@@ -0,0 +1,84 @@
+import { useEffect, useState } from 'react'
+import fetch from 'isomorphic-fetch'
+
+const useAPICall = <R>(url: string) => {
+  const [ready, setReady] = useState(false)
```

Comment:
so, do we cache results? for how long?

Reply:
It's cached on our proxy server side, as it was before.
Here it is: 
https://github.com/iterative/dvc.org/blob/blog-merging/middleware/api/comments.js#L18
https://github.com/iterative/dvc.org/blob/blog-merging/middleware/api/discourse.js#L8
https://github.com/iterative/dvc.org/blob/blog-merging/middleware/api/github.js#L6